packages/import/src/importIssues.ts

```diff
@@ -385,3 +360,116 @@ const createIssueWithRetries = async (
     }
   }
 };
+
+const handleLabels = async (
+  client: LinearClient,
+  data: ImportResult,
+  teamId: string,
+  state: {
+    existingLabelMap: { [name: string]: string };
+    existingLabelGroupsMap: { [name: string]: string };
+    existingGroupIdLabelMap: { [id: Id]: { [name: string]: Id } };
+  }
+) => {
+  const { existingLabelMap, existingLabelGroupsMap, existingGroupIdLabelMap } = state;
+  const { labels } = data;
+
+  const createLabel = async ({
+    name,
+    description,
+    color,
+    parentId,
+  }: {
+    name: string;
+    description?: string;
+    color?: string;
+    parentId?: string;
+  }) => {
+    const labelResponse = await client.createIssueLabel({
+      name,
+      description,
+      color,
+      teamId,
+      parentId,
+    });
+
+    const issueLabel = await labelResponse?.issueLabel;
+    return issueLabel?.id;
+  };
+
+  // Create labels and mapping to source data
+  const labelMapping = {} as { [id: string]: Id };
+
+  for (const labelId of Object.keys(labels)) {
+    const label = labels[labelId];
+    let labelName = _.truncate(label.name.trim(), { length: 80 });
```

Comment:
also updated this to 80, ref: https://github.com/linear/linear-app/blob/c871678388f2c5349b18aca1ff77f7b3dc484f0b/common/src/models/IssueLabelHelper.ts#L11-L12